formula/phplint.rb

```diff
@@ -0,0 +1,66 @@
+class Phplint < Formula
+  desc "Validator and documentator for PHP 5 and 7 programs"
+  homepage "http://www.icosaedro.it/phplint/"
+  url "http://www.icosaedro.it/phplint/phplint-3.0_20160307.tar.gz"
+  version "3.0-20160307"
+  sha256 "7a361166d1a6de707e6728828a6002a6de69be886501853344601ab1da922e7b"
+
+  depends_on "php@7.1"
+
+  def install
+    inreplace "php", "/opt/php/bin/php", "#{Formula["php@7.1"].opt_bin}/php"
+    inreplace "phpl", "$__DIR__/", "$__DIR__/../"
+    inreplace "phplint.tcl", "\"MISSING_PHP_CLI_EXECUTABLE\"", "#{opt_bin}/php"
+    inreplace "phplint.tcl", "set opts(phplint_dir) [pwd]", "set opts(phplint_dir) #{opt_prefix}"
+
+    prefix.install "modules"
```

Comment:
Can we install these in `libexec` instead? Also when you're installing multiple things in the same path, you can use commas

```
foo.install "this", "that"
```

Reply:
We can but we would need to `inreplace` some code such as
```
--modules-path "$__DIR__/modules" \
```

Reply:
I think if you put everything in libexec including the bin stuff and then use symlinks into `bin` that won't be needed.